gravitee-apim-gateway/gravitee-apim-gateway-reactor/src/main/java/io/gravitee/gateway/reactor/handler/impl/DefaultAcceptorResolver.java

```diff
@@ -16,32 +16,34 @@
 package io.gravitee.gateway.reactor.handler.impl;
 
 import io.gravitee.gateway.api.ExecutionContext;
-import io.gravitee.gateway.reactor.handler.EntrypointResolver;
-import io.gravitee.gateway.reactor.handler.HandlerEntrypoint;
+import io.gravitee.gateway.reactor.handler.AcceptorResolver;
+import io.gravitee.gateway.reactor.handler.HttpAcceptorHandler;
 import io.gravitee.gateway.reactor.handler.ReactorHandlerRegistry;
-import org.springframework.beans.factory.annotation.Autowired;
 
 /**
  * @author David BRASSELY (david.brassely at graviteesource.com)
  * @author GraviteeSource Team
  */
-public class DefaultEntrypointResolver implements EntrypointResolver {
+public class DefaultAcceptorResolver implements AcceptorResolver {
 
+    // TODO This attribute should be renamed/removed as entrypoint has been renamed to Acceptor
     public static final String ATTR_ENTRYPOINT = ExecutionContext.ATTR_PREFIX + "entrypoint";
+    public static final String ATTR_ACCEPTOR = ExecutionContext.ATTR_PREFIX + "acceptor";
 
     private final ReactorHandlerRegistry handlerRegistry;
 
-    public DefaultEntrypointResolver(ReactorHandlerRegistry handlerRegistry) {
+    public DefaultAcceptorResolver(ReactorHandlerRegistry handlerRegistry) {
         this.handlerRegistry = handlerRegistry;
     }
 
     @Override
-    public HandlerEntrypoint resolve(ExecutionContext context) {
-        for (HandlerEntrypoint entrypoint : handlerRegistry.getEntrypoints()) {
-            if (entrypoint.accept(context.request())) {
-                context.setAttribute(ATTR_ENTRYPOINT, entrypoint);
+    public HttpAcceptorHandler resolve(ExecutionContext context) {
+        for (HttpAcceptorHandler acceptorHandler : handlerRegistry.getHttpAcceptorHandlers()) {
+            if (acceptorHandler.accept(context.request())) {
+                context.setAttribute(ATTR_ENTRYPOINT, acceptorHandler);
+                context.setAttribute(ATTR_ACCEPTOR, acceptorHandler);
```

Comment:
This is a new one and we keep old one ATTR_ENTRYPOINT for compatibility.

Reply:
Yes, I just asking if, because it is a new one, we should put it in internalAttributes instead of attributes (attributes are more related to attributes added by the user) ?